editor/plugins/tiles/tile_atlas_view.cpp

```diff
@@ -286,6 +298,16 @@ void TileAtlasView::_draw_base_tiles() {
 	}
 }
 
+void TileAtlasView::_clear_material_canvas_items() {
+	for (KeyValue<Ref<Material>, RID> kv : material_tiles_draw) {
+		if (kv.value == base_tiles_draw->get_canvas_item()) {
```

Comment:
I guess this cannot happen, I don't think you add `base_tiles_draw->get_canvas_item()` to the list

Reply:
I do, for the empty material.